DentOS_Framework/DentOsTestbed/src/dent_os_testbed/test/test_suite/functional/vlan/test_vlan_setup.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+from dent_os_testbed.lib.ip.ip_link import IpLink
+from dent_os_testbed.lib.bridge.bridge_vlan import BridgeVlan
+
+pytestmark = pytest.mark.suite_vlan_functioning
+
+
+@pytest.mark.asyncio
+async def test_vlan_simple_set_up(testbed):
+    """
+    Test Name: VLAN setup: 4 links 2 VLANs
+    Test Suite: suite_vlan_functioning
+    Test Overview: Test VLAN setup
+    Test Procedure:
+    1. Configure a VLAN-aware bridge with 4 links
```

Comment:
You need to add more details. E.g. what ports and what VLANs should be configured in details.